resources/views/celebrities/show.blade.php

```diff
@@ -0,0 +1,35 @@
+<x-app-layout>
+    <x-slot name="header">
+        Celebrity Profile
```

Comment:
Maybe it is worth showing the Celebrity name in the header, instead of Celebrity Profile?